src/main/scala/cognite/spark/v1/RelationshipsRelation.scala

```diff
@@ -0,0 +1,191 @@
+package cognite.spark.v1
+
+import java.time.Instant
+
+import cats.effect.IO
+import cats.implicits._
+import cognite.spark.v1.PushdownUtilities.{
+  cogniteExternalIdSeqToStringSeq,
+  externalIdsToContainsAny,
+  getExternalIdSeqFromExternalId,
+  idsFromWrappedArray,
+  pushdownToParameters,
+  shouldGetAll,
+  stringSeqToCogniteExternalIdSeq,
+  timeRangeFromMinAndMax,
+  toPushdownFilterExpression
+}
+import cognite.spark.v1.SparkSchemaHelper.{asRow, fromRow, structType}
+import com.cognite.sdk.scala.v1._
+import fs2.Stream
+import org.apache.spark.sql.sources.{Filter, InsertableRelation}
+import org.apache.spark.sql.types._
+import org.apache.spark.sql.{Row, SQLContext}
+
+class RelationshipsRelation(config: RelationConfig)(val sqlContext: SQLContext)
+    extends SdkV1Relation[RelationshipsReadSchema, String](config, "relationships")
+    with InsertableRelation
+    with WritableRelation {
+  import CdpConnector._
+
+  override def schema: StructType = structType[RelationshipsReadSchema]
```

Comment:
Schema is not created for "Relationship" because of the reason mentioned below. Interface to user will be RelationshipsReadSchema.